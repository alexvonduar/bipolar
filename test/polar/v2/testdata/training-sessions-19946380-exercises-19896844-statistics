
Uw¡
…‡@?5A"»©5C]K¸C¦+D:33«A•}ÆAffB