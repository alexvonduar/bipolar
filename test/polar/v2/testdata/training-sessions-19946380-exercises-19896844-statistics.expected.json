{
    "altitude": [
        {
            "average": [
                368.58877563476562
            ],
            "maximum": [
                528.6820068359375
            ],
            "minimum": [
                181.66300964355469
            ]
        }
    ],
    "heartrate": [
        {
            "average": [
                119
            ],
            "maximum": [
                161
            ],
            "minimum": [
                85
            ]
        }
    ],
    "speed": [
        {
            "2": [
                "3f350641"
            ],
            "maximum": [
                4.219667911529541
            ]
        }
    ],
    "temperature": [
        {
            "average": [
                24.811319351196289
            ],
            "maximum": [
                32.349998474121094
            ],
            "minimum": [
                21.399999618530273
            ]
        }
    ]
}
